src/components/Resources/Resources.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from 'react-query';
+import PersonalMenu from '../PersonalMenu';
+import Search from '../Search';
+import { Grid, Typography } from '@material-ui/core';
+import { ResourceCard } from './ResourceCard';
+import { getResources } from '../../utils/queries';
+
+function Resources() {
+  const [searchValue, setSearchValue] = useState('');
+  const { isLoading, data, error } = useQuery([searchValue], getResources);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  const search = searchValue => {
+    setSearchValue(searchValue);
+  };
+
+  const { results, count } = data;
+
+  const renderResults = () => {
+    if (!results) return null; // to be on safe side in case API breaks
+    return (
+      <Grid container spacing={1}>
+        {results.length === 0 ? (
+          <Grid item lg={9}>
+            <Typography>No resources found</Typography>
+          </Grid>
+        ) : (
+          results.map(resource => (
+            <Grid item lg={3} key={resource.guid}>
+              <ResourceCard {...resource} />
+            </Grid>
+          ))
+        )}
+      </Grid>
+    );
+  };
+
+  return (
+    <Grid container spacing={1}>
+      <Grid item lg={3}>
+        <PersonalMenu />
+      </Grid>
+      <Grid item lg={9}>
+        <h2>Resources</h2>
+        <Search label="Search resources" search={search} />
+        {searchValue && (
+          <Typography>
+            You have searched for "<strong>{searchValue}</strong>" and gotten
+            <strong> {count}</strong> results.
+          </Typography>
+        )}
+        <br />
+        {error ? <div className="errorMessage">{error}</div> : renderResults()}
```

Comment:
I'd preffer for us to handle this as:
```suggestion
        { error && <div className="errorMessage">{error}</div> }
        { results && renderResults() }
```

Which should remove the `if (!results) { return null }` condition in the `renderResults` mehtod.****

Reply:
Also, I don't think we actually have the getResource methods returning error objects though, correct?

Reply:
> Also, I don't think we actually have the getResource methods returning error objects though, correct?

Hmm... I might be reading this wrong, but I thought the way the React Query API worked was that we only needed to return { data }, but we get immediate access to `error`, `isError`, `isLoading`, `data`, etc. 

For example, in this official example (https://codesandbox.io/s/github/tannerlinsley/react-query/tree/master/examples/pagination?file=/pages/index.js:516-529), `fetchProjects` only returns a destructured data, but we get access to all of the following: 

```
  const {
    status,
    resolvedData,
    latestData,
    error,
    isFetching,
  } = usePaginatedQuery(['projects', page], fetchProjects, {})
```